src/components/ShareTokenForm.js

```diff
@@ -36,12 +36,11 @@ const ShareTokenForm = () => {
 
   return (
     <form onSubmit={handleSubmit} className="form">
-      <label htmlFor="shareToken">Share token</label>
       <input
-        className="input"
+        className="input mb-4 sm:mb-0"
         type="text"
         id="shareToken"
-        placeholder="three word token"
+        placeholder="Enter 3 word token"
         onChange={(e) => setSelectedToken(e.target.value)}
         value={selectedToken}
```

Comment:
I guess it's good to have the label anyway for a11y reason, you can wrap the input with a label (without any text).

Reply:
I agree with Reda. Could we use an aria label instead or hide the label?